src/LanguageManager/LanguageListTable.php

```diff
@@ -0,0 +1,127 @@
+<?php # -*- coding: utf-8 -*-
+
+declare( strict_types = 1 );
+
+namespace Inpsyde\MultilingualPress\LanguageManager;
+
+use Inpsyde\MultilingualPress\Common\Labels;
+use Inpsyde\MultilingualPress\Database\Table\LanguagesTable;
+
+class LanguageListTable extends \WP_List_Table
+{
+	/**
+	 * @var string
+	 */
+	protected $screen;
+
+	/**
+	 * Used in inherited method display()
+	 *
+	 * @var array
+	 */
+	protected $_args = [
+		'plural'   => '',
+		'singular' => '',
+		'ajax'     => false,
+		'screen'   => null,
+	];
+
+	/**
+	 * List of languages.
+	 *
+	 * We use "items" here for compatibility with the extended class.
+	 *
+	 * @var array
+	 */
+	public $items = [];
+
+	/**
+	 * @var array column headers
+	 */
+	public $_column_headers;
+
+	/**
+	 * @var array columns
+	 */
+	public $columns;
+
+	/**
+	 * LanguageListTable constructor.
+	 *
+	 * @param array  $languages
+	 * @param Labels $labels
+	 */
+	public function __construct( array $languages, Labels $labels )
+	{
+		$this->items   = $languages;
+		$this->columns = $labels->all();
+		$this->_args['screen'] = LanguageManagerSettingsPageView::CURRENT_SCREEN;
```

Comment:
I don't understand why you manage `_args` yourself, when you pass data to the parent constructor - which takes care of that... All arguments that you set can be passed to the parent constructor.